### PR TITLE
[pre-commit check-json] Align the entire repo with check-json

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-8236)

## Description
In order to run `pre-commit` on our entire repo, it is necessary to fix certain files that fail the `check-json` hook.

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/5_1_3.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Classifiers
+
+##### Cortex XDR - IR
+
+Remove duplicate key from the classifier.
```
Comment (bziser) [inline Packs/CortexXDR/ReleaseNotes/5_1_3.md:6]:
```suggestion
Removed duplicate key from the classifier.
```

File: Packs/DuoAdminApi/ReleaseNotes/4_0_9.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Duo Modeling Rule
+
+Remove duplicate keys from the schema.
```
Comment (bziser) [inline Packs/DuoAdminApi/ReleaseNotes/4_0_9.md:6]:
```suggestion
Removed duplicate keys from the schema.
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/2_0_25.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### SaasSecurityModelingRules
+
+Remove duplicate keys from the schema.
```
Comment (bziser) [inline Packs/PrismaSaasSecurity/ReleaseNotes/2_0_25.md:6]:
```suggestion
Removed duplicate keys from the schema.
```

File: Packs/epo/ReleaseNotes/2_0_25.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Mcafee Epo Modeling Rule
+
+Remove duplicate keys from the schema.
```
Comment (bziser) [inline Packs/epo/ReleaseNotes/2_0_25.md:6]:
```suggestion
Removed duplicate keys from the schema.
```

**Review by bziser — APPROVED** [review]

**Comment by mmhw** [comment]
Reason for force merge:
No test for Modeling Rules
```
$ ./Tests/scripts/test_modeling_rules.sh || EXIT_CODE=$?
Testing Modeling Rules - Results will be saved to /builds/xsoar/content/artifacts/marketplacev2/modeling_rules_results.xml
[2023-09-05 18:02:17,562] [INFO] Log file location: /builds/xsoar/content/artifacts/marketplacev2/logs/demisto_sdk_debug.log
[2023-09-05 18:02:17,566] [INFO] You are using demisto-sdk 1.20.1.
[2023-09-05 18:02:17,567] [INFO] Modeling Rules directories to test: [PosixPath('/builds/xsoar/content/Packs/DuoAdminApi/ModelingRules/DuoModelingRule'), PosixPath('/builds/xsoar/content/Packs/epo/ModelingRules/McafeeEpoModelingRules'), PosixPath('/builds/xsoar/content/Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3')]
[2023-09-05 18:02:17,568] [INFO] <<<< Test Modeling Rule: Packs/DuoAdminApi/ModelingRules/DuoModelingRule >>>>
[2023-09-05 18:02:17,573] [WARNING] No test data file found for Packs/DuoAdminApi/ModelingRules/DuoModelingRule
[2023-09-05 18:02:17,574] [ERROR] Please create a test data file for Packs/DuoAdminApi/ModelingRules/DuoModelingRule and then rerun
[2023-09-05 18:02:17,574] [ERROR] Error testing modeling rule Packs/DuoAdminApi/ModelingRules/DuoModelingRule
demisto-sdk modeling-rules test Packs/DuoAdminApi/ModelingRules/DuoModelingRule
[2023-09-05 18:02:17,574] [INFO] <<<< Test Modeling Rule: Packs/epo/ModelingRules/McafeeEpoModelingRules >>>>
[2023-09-05 18:02:17,577] [WARNING] No test data file found for Packs/epo/ModelingRules/McafeeEpoModelingRules
[2023-09-05 18:02:17,577] [ERROR] Please create a test data file for Packs/epo/ModelingRules/McafeeEpoModelingRules and then rerun
[2023-09-05 18:02:17,577] [ERROR] Error testing modeling rule Packs/epo/ModelingRules/McafeeEpoModelingRules
demisto-sdk modeling-rules test Packs/epo/ModelingRules/McafeeEpoModelingRules
[2023-09-05 18:02:17,577] [INFO] <<<< Test Modeling Rule: Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3 >>>>
[2023-09-05 18:02:17,580] [WARNING] No test data file found for Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3
[2023-09-05 18:02:17,580] [ERROR] Please create a test data file for Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3 and then rerun
[2023-09-05 18:02:17,580] [ERROR] Error testing modeling rule Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3
demisto-sdk modeling-rules test Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector_1_3
$ section_end "Test Modeling Rules"
[2023-09-05T18:02:17.934] section end
```

**Comment by eepstain** [comment]
@mmhw 
Please run a Bypass installation for those pack on an XSIAM tenant with a Stable version.

**Comment by mmhw** [comment]
> @mmhw Please run a Bypass installation for those packs on an XSIAM tenant with a Stable version.

Done successfully.

**Pull request merged**